data/mods/Xedra_Evolved/items/spell_learning_items/spell_learning_items_dreamer.json

```diff
@@ -217,7 +206,21 @@
     "material": [ "forged_dreamstuff" ],
     "symbol": "0",
     "color": "black",
-    "//scannable": [ false, "not a book, but shouldn't be scannable" ],
     "use_action": { "type": "learn_spell", "spells": [ "summon_winch" ] }
+  },
+  {
+    "id": "artifact_karma_arms",
+    "type": "GENERIC",
+    "category": "artifacts",
+    "name": "odd circle",
+    "description": "Circular unit of energy, with few another circles that almost make a connection around the center",
```

Comment:
```suggestion
    "description": "Circular unit of energy, surrounded by a few other circles that almost make a connection around the center.",
```
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/99621099/a55b7dc9-da2e-4237-b99a-a5755e294878)
So it looks like this?

Reply:
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/67688115/68aa67f9-4356-4ed2-876e-0d18eaee5b2e)